Sources/DeepLearning/Operators.swift

```diff
@@ -26,3 +26,326 @@ public func round<Scalar: BinaryFloatingPoint>(_ x: Tensor<Scalar>) -> Tensor<Sc
 public func identity<Scalar>(_ x: Tensor<Scalar>) -> Tensor<Scalar> {
     return x
 }
+
+//===----------------------------------------------------------------------===//
+// Normalization
+//===----------------------------------------------------------------------===//
+
+public extension Tensor where Scalar : TensorFlowFloatingPoint {
```

Comment:
We follow a slightly different convention here for colons: no space before colons. Could you replace all of them?

Reply:
done